modules/webxr/webxr_interface_js.cpp

```diff
@@ -714,7 +713,8 @@ void WebXRInterfaceJS::_update_input_source(int p_input_source_id) {
 
 			if (unlikely(hand_tracker.is_null())) {
 				hand_tracker.instantiate();
-				hand_tracker->set_hand(p_input_source_id == 0 ? XRHandTracker::HAND_LEFT : XRHandTracker::HAND_RIGHT);
+				hand_tracker->set_tracker_hand(p_input_source_id == 0 ? XRPositionalTracker::TRACKER_HAND_LEFT : XRPositionalTracker::TRACKER_HAND_RIGHT);
+				hand_tracker->set_tracker_name(p_input_source_id == 0 ? "/user/left" : "/user/right");
```

Comment:
Same note here regarding the names of the trackers now that the hand trackers are mixed in with the rest.

Reply:
Updated to match tracker names.